/**:
  ros__parameters:
    min_x: 0.0
    max_x: 4.0
    min_y: -0.5
    max_y: 1.0
    min_z: -0.9
    max_z: 1.0
    voxel_size_x: 0.2
    voxel_size_y: 0.2
    voxel_size_z: 0.2
    # D_voxel_size_x: 0.0015
    D_voxel_size_x: 0.04
    D_voxel_size_y: 0.04
    D_voxel_size_z: 0.04
    voxel_search_range: 2
    ball_radius: 0.1
    landmark_width: 0.9
    landmark_height: 0.91
    width_tolerance: 0.1
    height_tolerance: 0.4
    laser_weight: 10.0
    odom_weight_liner: 1.0e-2
    odom_weight_angler: 1.0e-2
    plane_iterations: 100
    line_iterations: 100